Readme.md

```diff
@@ -0,0 +1,390 @@
+# BillboardTopTen
+
+[BillboardTopTen][heroku] is a web application that allows music enthusiasts to explore Billboard's top ten charts throughout history. Inspired by [Billboard charts][Billboard], BillboardTopTen is built using Ruby on Rails on the backend, React.JS on the Front-end, and a PostgreSQL database.
```

Comment:
We don't actually use a PostgreSQL db, I'd remove. I'd add maybe a comment about crawling and scraping directly from Billboard's site.